### ISSUE TITLE
Bug when combining heterogeneous hlls

### ISSUE THREAD

**Issue by dvizzini** [AdRoll/python-hll#3]
I'm getting the following message:

```
  File "/usr/local/lib/python3.8/site-packages/python_hll/hll.py", line 505, in union
    self._heterogenous_union(other)
  File "/usr/local/lib/python3.8/site-packages/python_hll/hll.py", line 647, in _heterogenous_union
    self._heterogeneous_union_for_non_empty_hll(other)
  File "/usr/local/lib/python3.8/site-packages/python_hll/hll.py", line 572, in _heterogeneous_union_for_non_empty_hll
    self._sparse_probabilistic_storage = deepcopy(other._sparse_probabilistic_storage)
AttributeError: 'HLL' object has no attribute '_sparse_probabilistic_storage'
```

This is being run on the  `python:3` docker image. Let me know what other info will be helpful. Unfortunately reproducing this but with public data is difficult.

Thank you.

**Comment by lemurchik** [comment]
https://github.com/AdRoll/python-hll/blob/master/python_hll/hll.py#L572

Both of HLL has type FULL, so correct assignment should be:
```python
self._probabilistic_storage = deepcopy(other._probabilistic_storage)
```

I will try to do a pull-request.

**Comment by JonathanAquino** [comment]
Thanks very much @lemurchik - I published 0.1.3 with your fix in pypi: https://pypi.org/project/python-hll/
